integrations/digital/outlook.js

```diff
@@ -0,0 +1,143 @@
+import moment from 'moment';
+import flatten from 'lodash/flatten';
+import { Client } from '@microsoft/microsoft-graph-client';
+
+import env from '../loadEnv';
+import { OAuth2Manager } from '../authentication';
+import { ACTIVITY_TYPE_DIGITAL, DIGITAL_CATEGORY_EMAIL, UNIT_ITEM } from '../../definitions';
+
+import { getActivitiesFromEmail } from './parsers/index';
+
+const config = {
+  label: 'Outlook',
+  description:
+    "free personal email service from Microsoft that doesn't scan your email for the purpose of serving you ads",
+  type: ACTIVITY_TYPE_DIGITAL,
```

Comment:
I believe this line is not needed - I suggest removing it.

Reply:
If I remove it, where is the integration going to show in the integrations pane? (Or is the list there built manually?)

Reply:
The list is built manually indeed.

Reply:
```suggestion
```